ScienceDirect.js

```diff
@@ -117,6 +108,46 @@ function getPDFLink(doc, onDone) {
 		}
 	}
 	
+	// Attempting to use json data, which might be available to construct a direct pdf link.
+	// A json data element might be available when using partnered institution network access.
+	// The same pdf url is created that would be available from the embedded pdf object,
+	// however the embedded object code may not always be available for parsing.
+	var jsondata = doc.querySelector('script[type="application/json"]').innerHTML;
+	if (jsondata) {
+		try {
+			jsondata = JSON.parse(jsondata);
+			Zotero.debug("Found json data");
+		}
+		catch {
```

Comment:
This `catch` needs to cover the whole block, not just the `JSON.parse()`. If the current catch triggered, `jsondata` wouldn't be set and the below lines would cause the entire translation to fail.